### ISSUE TITLE
git submodule update doesn't work: problems with list-ui ('public' folder)

### ISSUE THREAD

**Issue by AnthonyAkentiev** [LiskHQ/lisk#153]
Here is my trace:

```
ubuntu@ip-172-31-9-142:~/lisk$ git submodule update
Cloning into 'public'...
remote: Counting objects: 3660, done.
remote: Compressing objects: 100% (135/135), done.
remote: Total 3660 (delta 97), reused 0 (delta 0), pack-reused 3525
Receiving objects: 100% (3660/3660), 8.19 MiB | 4.81 MiB/s, done.
Resolving deltas: 100% (2754/2754), done.
Checking connectivity... done.
fatal: reference is not a tree: 1c33290c4033335138a903d204b5345c34101c54
Cloning into 'test/lisk-js'...
remote: Counting objects: 318, done.
remote: Compressing objects: 100% (57/57), done.
remote: Total 318 (delta 28), reused 0 (delta 0), pack-reused 257
Receiving objects: 100% (318/318), 431.58 KiB | 0 bytes/s, done.
Resolving deltas: 100% (179/179), done.
Checking connectivity... done.
Submodule path 'test/lisk-js': checked out 'a23996c0c6c76be2379ce42d6fe79ccad3d09a36'
Unable to checkout '1c33290c4033335138a903d204b5345c34101c54' in submodule path 'public'
```

**Comment by fix** [comment]
do you have rights to create a folder public and write in it?

**Comment by AnthonyAkentiev** [comment]
Yes

**Comment by fix** [comment]
I remember i had this issue, but don't remember how i fixed it...

**Comment by fix** [comment]
can't reproduce anymore. 
Did you `git submodule init` before?

**Comment by fix** [comment]
oh wait just reproduced

**Comment by fix** [comment]
ok will fix it, in the between the workaround is
```
rm -fr public/.git/
git clone https://github.com/LiskHQ/lisk-ui.git public/
```

**Comment by karmacoma** [comment]
Fixed here #173.
